lib/manager/npm/post-update/index.ts

```diff
@@ -441,6 +441,10 @@ export async function getAdditionalFiles(
       './others/yarn',
       'YARN_CACHE_FOLDER'
     ),
+    YARN_GLOBAL_FOLDER: await ensureCacheDir(
+      './others/berry',
+      'YARN_GLOBAL_FOLDER'
```

Comment:
@rarkins should we ignore predefined env, we will do it in v26 anyways. 🤔

Reply:
I don't mind, keep it as is if that's simplest